src/markdown-pages/landing-ar.md

```diff
@@ -4,33 +4,35 @@ templateKey: landing-page
 slug: home
 pageTitle: ar Project | Home
 hero:
-  image: ar hero-img.jpg
+  image: hero-img.jpg
```

Comment:
We can actually delete hero.image and hero.imageDescription from all markdown files and from the GraphQL query. The hero background image is set in the theme

Reply:
Done. Thanks for reminding me to change the GraphQL query, too. I'm assuming there is no`alt` text?